packages/format-library/src/text-color/index.js

```diff
@@ -89,7 +89,6 @@ function TextColorEdit( {
 					value={ value }
 					onChange={ ( ...args ) => {
```

Comment:
Just a clean-up suggestion, doesn't effect anything:
```suggestion
					onChange={ onChange }
```
And remove lines 91 and 92.